mammoth/opts.py

```diff
@@ -1235,13 +1237,13 @@ def translate_opts(parser, dynamic=False):
     _add_logging_opts(parser, is_train=False)
 
     group = parser.add_argument_group('Efficiency')
-    group.add('--batch_size', '-batch_size', type=int, default=30, help='Batch size')
+    group.add('--batch_size', '-batch_size', type=int, default=300, help='Batch size')
     group.add(
         '--batch_type',
         '-batch_type',
-        default='sents',
+        default='tokens',
         choices=["sents", "tokens"],
-        help="Batch grouping for batch_size. Standard is sents. Tokens will do dynamic batching",
+        help="Batch grouping for batch_size. Standard is tokens (max of src and tgt). Sents is unimplemented.",
```

Comment:
so no support for max len padding, which is the thing that seems to work the least bad for now?